build_tools/build_detect_platform

```diff
@@ -485,6 +485,8 @@ if test -z "$PORTABLE"; then
   elif test -n "`echo $TARGET_ARCHITECTURE | grep ^arm`"; then
     # TODO: Handle this with approprite options.
     COMMON_FLAGS="$COMMON_FLAGS"
+  elif [ "$TARGET_OS" == IOS ]; then
+    COMMON_FLAGS="$COMMON_FLAGS"
```

Comment:
does "-march=native" not work on iOS? It enables important cpu features, like fast crc, if the cpu supports it

Reply:
It doesn't work for ARM before gcc-7.